micro-infra-spring-base/src/test/groovy/com/ofg/infrastructure/web/resttemplate/MetricsAspectSpec.groovy

```diff
@@ -23,6 +23,8 @@ class MetricsAspectSpec extends Specification {
             'http://db:1000/data.xml'              || 'db.1000.data_xml'
             'http://db/data.xml'                   || 'db.80.data_xml'
             'http://db:6006/data?foo=bar'          || 'db.6006.data'
+            'http://192.168.1.100:6006'            || '192.168.1.100.6006'
+            'http://192.168.1.100:6006/ticket/pay' || '192.168.1.100.6006.ticket.pay'
```

Comment:
Test for HTTPS added. No, we don't support IPv6, this test fails miserably:

    'http://[2001:db8:a0b:12f0::1]:21/api' || '2001.db8.a0b.12f0.1.21.api'

Don't think it's an issue at the moment. If it is, should go through separate issues one day.